app/app.js

```diff
@@ -69,25 +69,6 @@ if (module.hot) {
   });
 }
 
-// Chunked polyfill for browsers without Intl support
-if (!window.Intl) {
-  new Promise(resolve => {
-    resolve(import('intl'));
-  })
-    .then(() =>
-      Promise.all([
-        import('intl/locale-data/jsonp/en.js'),
-        import('intl/locale-data/jsonp/de.js'),
-      ]),
-    ) // eslint-disable-line prettier/prettier
-    .then(() => render(translationMessages))
-    .catch(err => {
-      throw err;
-    });
-} else {
-  render(translationMessages);
```

Comment:
Hi @mhatvan , thanks for your work.

I tested this PR; removing ther `render()` in line 88 makes the application not load for the first time, although if you make any change in the source code the HMR kicks in and the app renders correctly.  This affects the production build, that doesn't render the application at all. 

_Please, note that I'm working on a fork, but this should have the same results in the template_

Hope this helps with the PR.

Reply:
Hey, thank you for your review!

You are totally right and I am glad you spotted this, I accidentally removed the `render()` when removing the check for `window.Intl`, I re-added it again in the newest commit.

Reply:
Happy to help,

Are you sure we can remove `window.Intl` safely ? Checking [Can I use](https://caniuse.com/#feat=intl-pluralrules) and [Mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/PluralRules) seems like not all the major browsers support `window.Intl.PluralRules`. Not sure if this applies. 

Bests !

Reply:
The new `@formatjs/intl-pluralrules` dependency should take care of that.

Check up in the react-intl upgrade guide: https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#migrate-to-using-native-intl-apis

And you can read the discussion here: https://github.com/react-boilerplate/react-boilerplate/pull/2721#issuecomment-534292622